tree/ntuple/v7/test/ntuple_storage_daos.cxx

```diff
@@ -7,17 +7,20 @@ TEST(RPageStorageDaos, Basics)
 {
    ROOT::TestSupport::CheckDiagsRAII diags;
    diags.optionalDiag(kWarning, "in int daos_init()", "This RNTuple build uses libdaos_mock. Use only for testing!");
-   diags.requiredDiag(kWarning, "ROOT::Experimental::Detail::RPageSinkDaos::RPageSinkDaos", "The DAOS backend is experimental and still under development.", false);
+   diags.requiredDiag(kWarning, "ROOT::Experimental::Detail::RPageSinkDaos::RPageSinkDaos",
+                      "The DAOS backend is experimental and still under development.", false);
    diags.requiredDiag(kWarning, "[ROOT.NTuple]", "Pre-release format version: RC 1", false);
 
-   std::string daosUri("daos://" R__DAOS_TEST_POOL "/container-test-1");
+   std::string containerLabel("container-test-1");
+   std::string daosUri("daos://" R__DAOS_TEST_POOL "/");
+   daosUri += containerLabel;
```

Comment:
Why are we changing this w.r.t. the line before?  Compile-time concatenation of string literals is guaranteed by the standard (and simpler to read :slightly_smiling_face:).
```suggestion
   std::string daosUri("daos://" R__DAOS_TEST_POOL "/container-test-1");
```
(same for the other occurrences in this file)